intyg-proxy-service/integration-fakehsa/src/main/java/se/inera/intyg/intygproxyservice/integration/fakehsa/repository/BootstrapHsaService.java

```diff
@@ -0,0 +1,80 @@
+package se.inera.intyg.intygproxyservice.integration.fakehsa.repository;
+
+import static se.inera.intyg.intygproxyservice.integration.api.constants.HsaConstants.FAKE_HSA_PROFILE;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import jakarta.annotation.PostConstruct;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Profile;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.intygproxyservice.integration.fakehsa.repository.model.ParsedCareProvider;
+import se.inera.intyg.intygproxyservice.integration.fakehsa.repository.model.ParsedCredentialInformation;
+import se.inera.intyg.intygproxyservice.integration.fakehsa.repository.model.ParsedHsaPerson;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+@Profile(FAKE_HSA_PROFILE)
+public class BootstrapHsaService {
```

Comment:
Maybe add some unit test for this class. You can look at the bootstrap service for fakepu